pkg/edgeview/src/system.go

```diff
@@ -1215,7 +1233,7 @@ func runTechSupport(cmds cmdOpt, isLocal bool) {
 	closePipe(true)
 
 	printTitle("\n       - system info -\n\n", colorRED, false)
-	runSystem(cmds, "hw,model,pci,usb,lastreboot,newlog,volume,app,datastore,cipher,dmesg,configitem")
+	runSystem(cmds, "hw,model,pci,usb,lastreboot,newlog,volume,app,datastore,cipher,dmesg,configitem,pprof")
```

Comment:
This should not be in techsupport, since without option input it will just return, and it will not generate any output into the techsupport file.

Reply:
removed it